accepted/2020/net5/net5.md

```diff
@@ -0,0 +1,790 @@
+* Provide compatibility with existing concepts and NuGet packages
+* It would be great to get this into early previews of .NET 5
+* Support multi-targeting different versions of the same OS
+* Don't force multi-targeting for different versions of the same OS. It should
+  be possible to produce a single binary that can use newer APIs when calls are
+  properly guarded with an OS check.
+
+### Non-Goals
+
+* Replace TFMs or expand runtime identifiers (RIDs)
+
+## Design
+
+We'll have the following TFMs:
+
+| TFM             | Compatible With                                            | Comments                          |
+|-----------------|------------------------------------------------------------|-----------------------------------|
+| net5.0          | net1..4 (with NU1701 warning)                              | No WinForms or WPF                |
+|                 | netcoreapp1..3.1 (warning when WinForms/WPF is referenced) |                                   |
+|                 | netstandard1..2.1                                          |                                   |
+| net5.0-android  | xamarin.android                                            |                                   |
+|                 | (+everything else inherited from net5.0)                   |                                   |
+| net5.0-ios      | xamarin.ios                                                |                                   |
+|                 | (+everything else inherited from net5.0)                   |                                   |
+| net5.0-macos    | xamarin.mac                                                |                                   |
+|                 | (+everything else inherited from net5.0)                   |                                   |
+| net5.0-tvos     | xamarin.tvos                                               |                                   |
+|                 | (+everything else inherited from net5.0)                   |                                   |
+| net5.0-watchos  | xamarin.watchos                                            |                                   |
+|                 | (+everything else inherited from net5.0)                   |                                   |
+| net5.0-windows  | netcoreapp1..3.1                                           | WinForms + WPF                    |
+|                 | (+everything else inherited from net5.0)                   |                                   |
+| Tizen, Unity... | Will follow the Xamarin model                              |                                   |
+
+_**Open Issue**. In case of Xamarin, there is a discussion of whether the
+convergence with .NET Core will be a breaking change for existing NuGet packages
+(for example, there is a desire to change their namespaces). In that case, we
+should not map it to the existing TFM as that would be pointless._
+
+_**Open Issue**. Will the above scheme work for iPad?_
```

Comment:
So far iPadOS is a marketing (only) term - the same SDK/tools/libraries are being used.
That might change in the future... but it's tricky (even for Apple) to migrate projects (single binary) built for both _OS_